plugins/services/src/js/pages/EditFrameworkConfiguration.js

```diff
@@ -0,0 +1,143 @@
+import React, { PropTypes } from "react";
+import mixin from "reactjs-mixin";
+import { StoreMixin } from "mesosphere-shared-reactjs";
+import { routerShape } from "react-router";
+import deepEqual from "deep-equal";
+import CosmosPackagesStore from "#SRC/js/stores/CosmosPackagesStore";
+import UniversePackage from "#SRC/js/structs/UniversePackage";
+import FrameworkConfiguration from "#SRC/js/components/FrameworkConfiguration";
+import Loader from "#SRC/js/components/Loader";
+import RequestErrorMsg from "#SRC/js/components/RequestErrorMsg";
+
+class EditFrameworkConfiguration extends mixin(StoreMixin) {
```

Comment:
🎨 I'm ok with the current implementation as it's inline with many of our components but moving forward please try to not use the `StoreMixin` or mixins in gernal. Mixins are super hard reason about and debug — using them is an anti-pattern and we should always use composition instead. /cc @dcos/frontend-foundation

Reply:
Sounds good thanks for bring to attention 👍